### ISSUE TITLE
Update for the new Slack UI

### ISSUE THREAD

**Issue by cbullard-dev** [catppuccin/slack#10]
**Issue**
The currently available codes no longer work with the Slack UI since the design change

The colour codes will need to be updated to use only four Hex Colour Codes instead of the previous ten

<img width="300" alt="SCR-20241127-ipkj" src="https://github.com/user-attachments/assets/dbf41c3f-3c9b-4a30-964a-1e7825daa229">

<img width="300" alt="SCR-20241127-iphy" src="https://github.com/user-attachments/assets/c11601bb-7dce-4253-85ab-32c5e38af1c6">

Unfortunately I'm terrible with colours and design and I can't seem to make this work no matter what I do ☹️

**Comment by nithin-mk** [comment]
The "Paste your legacy theme colours" option is not appearing for me :(

**Comment by medeirosthiago** [comment]
You can paste it to a chat, send it to yourself on slack, it will appear an *Apply Slack Theme* button.
![image](https://github.com/user-attachments/assets/092dc5cb-adb3-4549-8352-6dcb926d3409)

**Comment by sgoudham** [comment]
Hey 👋,

You should be able to click on the "Paste your legacy theme colours" link and paste the 10 theme codes which should automatically map to the 4 colours. However, if that option has also been removed then I suppose we just need to give 4 colours directly

I have a feeling Slack is going to stop allowing custom colours all together... at some point.

**Comment by AncientRo** [comment]
@sgoudham Using "Paste your legacy theme colors" does not make the Slack UI look like in the screenshots. But maybe it is no longer supported to make it look like that.

Is there a known workaround, or should we give up on Catppuccin for Slack?

It is weird for them to remove customization options, not sure what the benefits of that are.

**Comment by sgoudham** [comment]
> Using "Paste your legacy theme colors" does not make the Slack UI look like in the screenshots. But maybe it is no longer supported to make it look like that.

Yeah, the screenshots haven't been updated so it won't look like that anymore. Pasting in a custom theme string will just map closely to Slack's built-in colours afaik.

> Is there a known workaround, or should we give up on Catppuccin for Slack?
>
> It is weird for them to remove customization options, not sure what the benefits of that are.

I can't speak to why they've simplified it, but yeah there will most likely be a point in time where you can't import a theme string at all. This is what I use at the moment:

<img width="528" alt="Image" src="https://github.com/user-attachments/assets/fdfab7db-a1f3-4ae4-8879-eaed0e667354" />

**Comment by pvinis** [comment]
to make it easier, heres mine too, which is what came from the readme plus @sgoudham's green. it should work for both light and dark modes.

- copy `#EFF1F5,#8839EF,#A6E3A1,#E64553`
- paste this in a dm to yourself on slack
- press apply
